docs/getting_started.md

```diff
@@ -88,7 +88,7 @@ export default compose(
 
 ### Decorators
 
-They are completely optional, but ES7 Decorators can be used. [The Simple Example](examples/simple) shows implementation without decorators, while [the Decorators Example](examples/decorators) shows the same application with decorators implemented.
+They are completely optional, but ES7 Decorators can be used. [The Simple Example](https://github.com/prescottprue/react-redux-firebase/blob/master/examples/complete/simple/src/Home.js) shows implementation without decorators, while [the Decorators Example](https://github.com/prescottprue/react-redux-firebase/blob/master/examples/snippets/decorators/App.js) shows the same application with decorators implemented.
```

Comment:
Great catch with these. We should be able to just add the preceding slash though right?

Maybe we should keep the references to the README of those examples and explain more in the readme?

Reply:
No, I tried adding slashes but it didn't work. Even some other relative links that are not mentioned in the Readme of the docs are also broken.